### PR TITLE
Feature: load mov as image background add audio in sequencer only no audio yet

### PR DESCRIPTION
**Pull request by kaamaurice**
## Changelog Description
Loading mov as image background check if any sound in sequencers to add audio as sound sequence in an empty channel.

### CODE REVIEW

File: openpype/hosts/blender/plugins/load/load_image.py [hunk 1]
```diff
@@ -169,12 +169,24 @@ def load(self, *args, **kwargs):
...
-            # Append audio in the sequencer
-            sound_seq = bpy.context.scene.sequence_editor.sequences.new_sound(
-                img.name, img.filepath, 1, bpy.context.scene.frame_start
-            )
-
-            # Add container metadata to sound
-            sound_seq.sound[AVALON_PROPERTY] = container.get(AVALON_PROPERTY)
+            # Append audio in the sequencer only if there is no sound yet
+            sequences = bpy.context.scene.sequence_editor.sequences
+            if all({seq.type != "SOUND" for seq in sequences}):
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/load/load_image.py]: no need for setting up a `set`, a strict generator works, you can remove the brackets `{...}`

File: openpype/hosts/blender/plugins/load/load_image.py [hunk 2]
```diff
@@ -169,12 +169,24 @@ def load(self, *args, **kwargs):
...
+                # Find an empty channel
+                used_channels = {seq.channel for seq in sequences}
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/load/load_image.py]: This variable is useless. You can move the built set to the `next` matching.

**Review by Tilix4 — CHANGES_REQUESTED** [review]
Nice! This is a much better design. Few optim

File: openpype/hosts/blender/plugins/load/load_image.py (hunk 2 above)
Comment (kaamaurice) [reply openpype/hosts/blender/plugins/load/load_image.py]: done

**Review by Tilix4 — APPROVED** [review]

**Pull request merged**